src/components/App.stories.js

```diff
@@ -79,6 +79,7 @@ export default {
           {code: 'en', name: 'English'},
         ]),
         mockLanguageChoicePut,
+        mockGetAnalyticsToolConfig(),
```

Comment:
looking at the other functions, this should be named `mockAnalyticsToolsConfigGet` where the HTTP method comes at the end :)